.gitignore

```diff
@@ -2,9 +2,6 @@ node_modules/
 public/
 .cache/
 .DS_Store
-static/*
```

Comment:
We don't want to accidentally commit all of the statics accidentally after building, can we special case this one file and keep the existing gitignore rules?

Reply:
Going to message you about this - have a question about why this is necessary